src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -107,7 +118,7 @@ const ExpandedNode: FC<ExpandedNodeProps> = memo(
           pathwayState={pathwayState}
           documentation={documentation}
           setComments={setComments}
-          comments={note?.notes ?? ''}
+          comments={notes ?? ''}
```

Comment:
I think changing this to 
```suggestion
          comments={notes ?? note?.notes ?? ''}
```
gives the desired behavior